checkov/terraform/graph_builder/graph_components/generic_resource_encryption.py

```diff
@@ -1,9 +1,16 @@
+from typing import Dict, List, Union, Tuple, Any
+
 from checkov.common.graph.graph_builder import EncryptionTypes
 from checkov.terraform.checks.utils.utils import decode_graph_property_value
 
 
 class GenericResourceEncryption:
-    def __init__(self, resource_type: str, attribute_values_map: dict, enabled_by_default=False):
+    def __init__(
+        self,
+        resource_type: str,
+        attribute_values_map: Dict[str, Union[List[bool], List[str]]],
```

Comment:
@gruebel Shouldn't it be `Dict[str, List[Union[bool, str]]]`?

Reply:
I checked and there shouldn't be a `list` with `bool` and `str` values mixed, just one or the other. But if you think this could happen, then I can adjust it.